anonymizer/OpenDiffix.Service/Program.fs

```diff
@@ -190,27 +190,25 @@ let handleExport
 
   let countColumn = $"diffix_count(%s{countInput}, %s{aidColumn}) AS count"
 
-  let query =
+  // We get a hold of the star bucket results reference via side effects.
+  let mutable suppressedAnonCount = Null
```

Comment:
The hook should be smart enough to ignore global queries, like Led does.

Reply:
The problem is the missing mandatory aggregator. 
I thought it is simpler to conditionally call the hook, rather than modify the hook to handle such cases.